addons/mail/static/tests/emoji/emoji.test.js

```diff
@@ -216,3 +216,19 @@ test("selecting an emoji while holding down the Shift key prevents the emoji pic
     await contains(".o-EmojiPicker");
     await contains(".o-mail-Composer-input", { value: "👺" });
 });
+
+test("shortcodes shown in emoji title in message", async () => {
+    const pyEnv = await startServer();
+    const channelId = pyEnv["discuss.channel"].create({ name: "" });
+    pyEnv["mail.message"].create({
+        res_id: channelId,
+        model: "discuss.channel",
+        body: "💑😇",
+        author_id: serverState.partnerId,
+    });
+    await start();
+    await openDiscuss(channelId);
+    await contains(".o-mail-Message", { text: "💑😇" });
+    await contains(".o-mail-Message *[title=':couple_with_heart:']");
+    await contains(".o-mail-Message *[title=':innocent: :halo:']");
```

Comment:
Isn't * not necessary in this case? Or if we know it is a span, we might as well specify it?

Reply:
Ideally the test should also check the correct text (emoji) is matching with the title.